src/components/list/_list.scss

```diff
@@ -103,15 +106,13 @@ $include-html: false !default;
     @include mint-list-basic-styles();
     display: inline-block;
     vertical-align: inherit;
-    font-size: $breadcrumbListFontSize;
-    line-height: $breadcrumbListFontSize * 1.5;
     color: $breadcrumbListSeparatorColor;
 
     &__element {
       display: inline-block;
 
       &:after {
-        content: '·';
+        content: '•';
```

Comment:
We can, but I assumed that changing the fonts means changing the elements like separators and links - as centering and proportions change as well.

Having this thing parameterized in style-guide means that we can change it anytime across the codebase with a simple change. 

If I would open an issue - it will require me to switch the context and made changes to the non-merged branch. Hence - postponing merging this thing.

Do you really want me to revert it?